### ISSUE TITLE
RimFridge compatibility broken

### ISSUE THREAD

**Issue by frame** [lilwhitemouse/RimWorld-LWM.DeepStorage#55]
Hi,

In believe support for RimFridge got broken because the RimFridge Mod got renamed from "[KV] RimFridge - 1.0" to "[KV] RimFridge - 1.1"

RimWorld-LWM.DeepStorage/_Mod/LWM.DeepStorage/Patches/AddNewUnits_RimFridge.xml
```
<Patch>
    <Operation Class="PatchOperationFindMod">
    <mods>
      <li>[KV] RimFridge - 1.0</li>
    </mods>
    [..]
```

Workshop Link: https://steamcommunity.com/sharedfiles/filedetails/?id=1180721235

(Another user also confirmed this in one of the comments on Steam)

**Comment by lilwhitemouse** [comment]
Oh.  Well THAT's easy enough to fix.

**Comment by lilwhitemouse** [comment]
Should be fixed for 1.1
